terraform.tfvars.json

```
@@ -0,0 +1,38 @@
+{
+  "repository": [
+    {
+      "name": "vscode",
+      "supermarket_name_override": "sc-vscode",
+      "repo_type": "cookbook"
+    },
+    {
+      "name": "foobar",
+      "repo_type": "cookbook",
+      "additional_status_checks": ["foo", "bar"]
+    },
+    {
+      "name": "tf-test",
+      "repo_type": "terraform",
+      "additional_topics": ["jason-testing"]
+    },
+    {
+      "name": "tf-test-with-url",
+      "repo_type": "terraform",
+      "additional_topics": ["jason-testing"],
+      "homepage_url_override": "google.com"
+    },
+    {
+      "name": "ide-test-with-url",
+      "repo_type": "ide",
```

```suggestion
      "type": "ide",
```

I think just "type" works here?

---

I prefer a more specific name as if a type property ever comes out of github I don't want a clash